java/src/main/java/ai/rapids/cudf/ColumnView.java

```diff
@@ -2323,6 +2323,49 @@ public static ColumnView makeStructView(ColumnView... columns) {
     return makeStructView(columns[0].rows, columns);
   }
 
+  /**
+   * Create a column of bool values indicating whether the specified scalar
+   * is an element of each row of a list column.
+   * Output `column[i]` is set to null if one or more of the following are true:
+   * 1. The key is null
+   * 2. The column vector list value is null
+   * 3. The list row does not contain the key, and contains at least
+   *    one null.
+   * @param key the scalar to look up
+   * @return a Boolean ColumnVector with the result of the lookup
+   */
+  public final ColumnVector listContains(Scalar key) {
+    assert type.equals(DType.LIST) : "column type must be a LIST";
+    try (ColumnView childCv = this.getChildColumnView(0)) {
```

Comment:
I am not thrilled with this check because getting the child column view is not free and even if assertions are disabled we will still end up getting it only to close it again.  If the underlying C++ code already does this check then there is no real need to do it. If it does not then we should have it be a separate private method that returns a boolean so if assertions are disabled we can remove the call entirely.

Reply:
Totally agree 😌 that's why I thought it was a bit ugly. @mythrocks if you agree, I'll take this out.

Reply:
Hmm... I had not considered that. Please go ahead and remove the check.

Reply:
Done. Please take another look